features/customers.feature

```diff
@@ -364,4 +364,90 @@ Scenario: Update a Customer with Missing Password
     And I should see "Required field" in the "Password" error string
     When I set the "Password" to "test"
     And I press the "Update" button
-    Then The "Password" error string should be gone 
\ No newline at end of file
+    Then The "Password" error string should be gone 
+
+Scenario: Delete a Customer that Exists
+    When I visit the "Home Page"
+    When I visit the "Home Page"
+    And I press the "Search" button
+    And I copy the "Id" field
+    And I press the "Clear" button
```

Comment:
Add this after pressing the clear button:
`Then I should see the message "Cleared"`